test/helpers/base-config.js

```diff
@@ -3,28 +3,29 @@
 var StyleLintPlugin = require('../../');
 var webpack = require('./webpack');
 
-var configFilePath = getPath('./.stylelintrc');
+module.exports = function (options) {
+  var configFilePath = getPath('./.stylelintrc');
+  const pluginOptions = Object.assign({ configFile: configFilePath }, options || {});
```

Comment:
This is a great change! Thank you!

However, could you:

- use the `object-assign` polyfill, to remain consistent with the rest of the codebase
- update its usages in the tests to remove the unnecessary `config` objects everywhere.